apps/judicial-system/backend/migrations/20240827132504-create-subpoena.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      queryInterface.createTable(
+        'subpoena',
+        {
+          id: {
+            type: Sequelize.UUID,
+            primaryKey: true,
+            allowNull: false,
+            defaultValue: Sequelize.UUIDV4,
+          },
+          created: {
+            type: 'TIMESTAMP WITH TIME ZONE',
+            defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'),
+            allowNull: false,
+          },
+          modified: {
+            type: 'TIMESTAMP WITH TIME ZONE',
+            defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'),
+            allowNull: false,
+          },
+          defendant_id: {
+            type: Sequelize.UUID,
+            references: {
+              model: 'defendant',
+              key: 'id',
+            },
+            allowNull: false,
+          },
+          case_id: {
+            type: Sequelize.UUID,
+            references: {
+              model: 'case',
+              key: 'id',
+            },
+            allowNull: true,
+          },
+          subpoena_id: {
+            type: Sequelize.STRING,
+            allowNull: true,
+          },
+          acknowledged: {
+            type: Sequelize.BOOLEAN,
+            allowNull: true,
+          },
+          acknowledged_date: {
+            type: 'TIMESTAMP WITH TIME ZONE',
+            allowNull: true,
+          },
+          comment: {
+            type: Sequelize.STRING,
```

Comment:
Consider changing the type to TEXT. String can only contain 256 characters.

Reply:
Good point, I'll change it so we don't need to add in another migration for that later